src/zulip.ts

```diff
@@ -6,7 +6,14 @@ import * as log from './logger';
 
 const PRESENCE_TICK = 140000;
 
-type Config = { username: string; apiKey: string; realm: string };
+type Config = {
+	username: string;
+	apiKey: string;
+	realm: string;
+	streamName: string;
+	meetUrl: string;
```

Comment:
This is mixing google meets with the Zulip module when they 2 independent modules. The bot.ts file should be the one gluing them.

Reply:
Looked at the code again, and the bot is the one gluing them. The only way the Zulip bot gets the meetUrl, which it needs for the topic message, is when `bot.ts` passes the config to Zulip on init: https://github.com/people-os/open-office-bot/pull/10/files#diff-6237e411e4b5750f5e3713c4739039620a279cdf9c8c596d573a97e87f1c8e0fR17-R23